pkg/process/config/config.go

```diff
@@ -379,36 +379,36 @@ func NewSystemProbeConfig(loggerName config.LoggerName, yamlPath string) (*Agent
 }
 
 func loadEnvVariables() {
-	for envKey, cfgKey := range map[string]string{
-		"DD_PROCESS_AGENT_CONTAINER_SOURCE": "process_config.container_source",
-		"DD_SCRUB_ARGS":                     "process_config.scrub_args",
-		"DD_STRIP_PROCESS_ARGS":             "process_config.strip_proc_arguments",
-		"DD_PROCESS_AGENT_URL":              "process_config.process_dd_url",
-		"DD_ORCHESTRATOR_URL":               "process_config.orchestrator_dd_url",
+	for _, variable := range []struct{ env, cfg string }{
```

Comment:
Would you please add a comment here about the order being important?  I see the note below about preferring DD_LOG_LEVEL over LOG_LEVEL but it would be good to have a note up top for visibility.

Reply:
Done!